### PR TITLE
Improve example for using when without an argument

### PR DESCRIPTION
**Pull request by AfzalivE**
In the current example:
Both branches are for x, it makes sense to use `when(x)` in it.

But if we switch the example to have two variables, then it makes sense to just use when without arguments.

### CODE REVIEW

**Comment by p7nov** [comment]
Hi!
Thanks for your PR.
How would you use `when(x)` for this case?

**Comment by AfzalivE** [comment]
You wouldn't. 

This is more to highlight a case where using when without an argument makes more sense over `when(x)`

File: pages/docs/reference/control-flow.md [hunk 1]
```diff
@@ -146,8 +146,8 @@ If no argument is supplied, the branch conditions are simply boolean expressions
 ```kotlin
 when {
     x.isOdd() -> print("x is odd")
-    x.isEven() -> print("x is even")
-    else -> print("x is funny")
+    y.isEven() -> print("y is even")
+    else -> print("x and y are both funny")
```
Comment (p7nov) [inline pages/docs/reference/control-flow.md:150]:
```suggestion
    else -> print("x+y is even.")
```

**Comment by p7nov** [comment]
Ok, I see. Just one more suggestion to make the last line meaningful.

**Comment by p7nov** [comment]
Thanks!

**Pull request merged**